### ISSUE TITLE
RuntimeError: Error(s) in loading state_dict for PointRCNN

### ISSUE THREAD

**Issue by WakyWakyWayne** [dmatos2012/pointrcnn_detector_ros#2]
RuntimeError: Error(s) in loading state_dict for PointRCNN:
	size mismatch for rpn.backbone_net.SA_modules.0.mlps.0.layer0.conv.weight: copying a param with shape torch.Size([16, 3, 1, 1]) from checkpoint, the shape in current model is torch.Size([16, 4, 1,1])
        size mismatch for rpn.backbone_net.SA_modules.0.mlps.1.layer0.conv.weight: copying a param with shape torch.Size([32, 3, 1, 1]) from checkpoint, the shape in current model is torch.Size([32, 4, 1, 1]).
	size mismatch for rpn.backbone_net.FP_modules.0.mlp.layer0.conv.weight: copying a param with shape torch.Size([128, 256, 1, 1]) from checkpoint, the shape in current model is torch.Size([128, 257, 1, 1]).

**Comment by dmatos2012** [comment]
HI Waky,Wayky,

Did you download the `PointRCNN.pth` linked in the README?

**Comment by WakyWakyWayne** [comment]
Yes.  I have done PointRCNN on Kitti, but I met this problem when I use your ros.

**Comment by gehadza** [comment]
set use_intentisity=False

**Comment by zhibingduan** [comment]
@gehadza Hello, I had the same problem and asked the gpt that it was due to a mismatch between the checkpoints and the model. You talk about changing the use_intensity=false, but I don't know where to change the value. By the way, if I retrain the model and run ros with the trained model, should that solve the problem. Any suggestions would be greatly appreciated!

**Comment by zhibingduan** [comment]
@gehadza @WakyWakyWayne Hello, I'm having the same problem and asked gpt about it saying it's due to a mismatch between the checkpoints and the model. You talk about changing use_intensity=false but I don't know where to change the value. By the way, if I retrain the model and run ros with the trained model, will it solve the problem? Any advice would be greatly appreciated!
